config/locales/gobierto_participation/views/es.yml

```diff
@@ -22,6 +22,29 @@ es:
           donde debatiremos en persona sobre los temas de este proceso. Si no puedes
           asistir, iremos publicando online todo lo que se hable en la reuniones para
           facilitar el seguimiento y hacer accesibles las conclusiones.
+    issues:
+      current_group:
+        current_stage: Fase actual
+      index:
+        intro: Tus ideas ayudan a mejorar %{location_name} y la vida de todos los
```

Comment:
Fix it!! https://github.com/PopulateTools/gobierto/pull/868/commits/f9b7849cd40f84a39111766616d065ded802053e